app/models/pickup.rb

```diff
@@ -0,0 +1,4 @@
+class Pickup < ActiveRecord::Base
```

Comment:
@AmirKremer do we need both Pickup and Review? What's the difference between the two?